doc/help/elektra-cascading.md

```diff
@@ -32,6 +32,12 @@ are available (in the meta data of respective `spec`-keys):
     the `fallback`-keys will be searched
 - `default`: this value will be used if nothing else was found
 
+They can be used like this:
+
+```
+$ kdb set /overrides/test "example override"
+$ sudo kdb setmeta spec/test override/#0 /overrides/test
+```
```

Comment:
Please use the same syntax as done for all other examples:
(space space, newline, then tab, then single `)